portfolio/src/main/webapp/script.js

```diff
@@ -93,14 +93,42 @@ function drawChart() {
   ]);
   const options = {
     'title': 'Zoo Animals',
+    'pieHole': 0.4,
     'width':500,
     'height':400
   };
-  const chart = new google.visualization.PieChart(document.getElementById('chart-container'));
+  const chart = new google.visualization.PieChart(document.getElementById('chart-div'));
   chart.draw(data, options);
 }
 
+function drawChart2() {
+  var data = new google.visualization.DataTable();
+  data.addColumn('number', 'Week');
+  data.addColumn('number', 'Stock A');
+  data.addColumn('number', 'Stock B');
+  data.addRows([
+    [1,  50, 70],
+    [2,  130, 100],
+  ]);
+  var options = {
+    chart: {
+      title: 'Weekly Performcance of Stock A and Stock B',
+      subtitle: 'in dollars (USD)'
+    },
+    width: 900,
+    height: 500
+  };
+  var chart = new google.charts.Line(document.getElementById('linechart_material'));
+  chart.draw(data, google.charts.Line.convertOptions(options));
+}
+
 function loadChartsApi() {
-  google.charts.load('current', {'packages':['corechart']});
+  google.charts.load('current', {'packages':['corechart', 'line']});
   google.charts.setOnLoadCallback(drawChart);
+  google.charts.setOnLoadCallback(drawChart2);
 }
+
+/*function loadChartsApi2() {
+  google.charts.load('current', {'packages':['line']});
+  google.charts.setOnLoadCallback(drawChart2);
+}*/
```

Comment:
nit: this commented out code should be removed

Reply:
Addressed in "remove commented code in script.js"